### PR TITLE
Adding new page, with modifications to the first.

### PR DESCRIPTION
**Pull request by frostyclawz**
Modifications to index.html and style.css here. Adding another page with index2.html and style2.html with images.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,7 +7,10 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Vincent's Portfolio</h1>  
+    <h1>Vincent's Portfolio</h1>
+    <div id="links">
+        <a href="index2.html">Gaming</a>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index.html:12]: Perhaps this should be called 'gaming.html' instead of 'index2'?

index2 doesn't really give any indication of what's in it

File: portfolio/src/main/webapp/index2.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Vincent Stevenson</title>
+        <link rel="stylesheet" href="style2.css">
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index2.html:6]: likewise, style2.css should be renamed to match the new name of this HTML file (so maybe gaming.css)

File: portfolio/src/main/webapp/index2.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            </p>
+            <ul>
+                <li><a href="#Destiny">Destiny</a></li>
+                <li><a href="#Smite">Smite</a></li>
+                <li><a href="#Spyro:_Reignited_Trilogy">Spyro: Reignited Trilogy</a></li>
+            </ul>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index2.html:24]: These ids should also be more descriptive. Div2 doesn't indicate much about what is in here, and it becomes confusing to read in the CSS as well

File: portfolio/src/main/webapp/index2.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Vincent Stevenson</title>
+        <link rel="stylesheet" href="style2.css">
+        <script src="script.js"></script>
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index2.html:7]: It looks like script.js isn't being used from this page -- if so, the script tag importing it should be removed

File: portfolio/src/main/webapp/index2.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                <li><a href="#Spyro:_Reignited_Trilogy">Spyro: Reignited Trilogy</a></li>
+            </ul>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
+            <a id="Destiny">
+                <h2>Destiny</h2>
+            </a>
+            <img src="/images/Destiny.jpg", width="900px">
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index2.html:28]: Make sure you credit the sources of these images (probably just adding a README file with their sources in the images folder is fine)

File: portfolio/src/main/webapp/index2.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+                games in an updated fashion was an amazing experience. I've always had a fondness
+                of older gamees, and getting to play a remake of of these games was simply...
+                awesome.
+            </P>
+            </br>
+            <p>Favorite Spyro Game?: Spyro 2</p>
+        </div>  
+        </br>
+        </br>  
+        <div id="link_row">
```
Comment (ianjjohnson) [inline portfolio/src/main/webapp/index2.html:81]: It looks like this opening <div> doesn't have a corresponding </div> anywhere

Comment (jakecarlson1) [reply portfolio/src/main/webapp/index2.html:81]: Did you just inject HTML into GitHub's comment view? I'm assuming you wrote `<div>` in your comment.

<script>alert("pls don't show up")</script>

Comment (jakecarlson1) [reply portfolio/src/main/webapp/index2.html:81]: Okay, glad that didn't fire an alert (sorry this has nothing to do with the CL, I just found this amusing).

File: portfolio/src/main/webapp/style2.css [hunk 7]
```diff
@@ -0,0 +1,85 @@
+h1 {
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/style2.css:1]: Is this file still needed with Home/Gaming.css?

File: portfolio/src/main/webapp/Home.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Home.html:1]: The indentation/spacing here is quite a bit different than the other HTML files. Can you please reformat this?

File: portfolio/src/main/webapp/Home.css [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    word-spacing: 2px;
+    color: #ffffff;
+    font-weight: bold;
+    text-decoration: underline solid white;
+    font-style: bold;
+    font-variant: normal;
+    text-transform: none;
+    text-shadow: 0 -1px 4px #FFF, 0 -2px 10px rgb(0, 238, 255), 0 -10px 20px #008cff, 0 -18px 40px rgb(195, 0, 255);
+    text-align: center;
+}
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Home.css:14]: It looks like this is the same style def from Gaming.css. Maybe move the duplicates styles to a `common.css` file?

File: portfolio/src/main/webapp/Gaming.css [hunk 10]
```diff
@@ -0,0 +1,87 @@
+h1 {
```
Comment (jakecarlson1) [inline portfolio/src/main/webapp/Gaming.css:1]: I think best practice is to have lower case file names for css/html files. Can you please rename Gaming* to gaming* and Home* to home*?
